lib/this_data/track_request.rb

```diff
@@ -11,12 +11,20 @@ class ThisDataTrackError < StandardError; end
 
     # Will pull request and user details from the controller, and send an event
     # to ThisData.
-    def thisdata_track(verb: ThisData::Verbs::LOG_IN)
+    # Arguments:
+    #   verb: (String, Required). Defaults to ThisData::Verbs::LOG_IN.
+    #   user: (Object, Optional). If you want to override the user record
+    #     that we would usually fetch, you can pass it here.
```

Comment:
You could add something here to describe that the normal behaviour is, just as a convenience so the developer. E.g. "Unless a user is specified here we'll attempt to get the user record as specified in the ThisData gem configuration. This defaults to `current_user`". Or something.